lib/octokit/connection.rb

```diff
@@ -160,7 +160,7 @@ def request(method, path, data, options = {})
     # @return [Boolean] True on success, false otherwise
     def boolean_from_response(method, path, options = {})
       request(method, path, options)
-      @last_response.status == 204
+      @last_response.status == 204 || @last_response.status == 201
```

Comment:
Ah, I see what you mean now regarding [your comment](https://platform.github.community/t/about-the-projects-rest-api-category/66/2?u=bswinnerton) in the Platform Community Discourse. I'll chime in on that thread shortly of our original intentions of returning a `201` here.